README.md

```diff
@@ -71,7 +71,7 @@
 
 > Add a link to your deployed project.
 
-- [Live Demo Link](https://shegy28.github.io/)
```

Comment:
- Please add in this ReadMe the instructions about where and how to generate any necessary keys to run the App
- My suggestions: `npm install`, `npm run build`, `npm start`. or other codes you may know ...